NERODevelopment/content/Pit.qml

```diff
@@ -33,7 +35,7 @@ Rectangle {
         color: "transparent"
 
         Rectangle {
```

Comment:
I think the best way to refactor this is to use just RowLayouts and ColumnLayouts, so basically one RowLayout, with Three ColumnLayouts in it, each of the column layouts will have two components in them. Thatll make the layout much cleaner.